File: stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/AppSettings.java
package io.getstream.chat.docs.java;

import android.content.Context;

import io.getstream.chat.android.client.ChatClient;

public class AppSettings {
    private Context context;

    /**
     * @see <a href="https://getstream.io/chat/docs/android/multi_region/?language=java#client-location-settings">Client Location Settings</a>
     */
    public void clientLocationSettings() {
        // Set the base URL
        ChatClient client = new ChatClient.Builder("{{ api_key }}", context)
                .baseUrl("https://chat.stream-io-api.com/")
                .build();
    }
}

File: stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/AppSettings.kt
package io.getstream.chat.docs.kotlin

import android.content.Context
import io.getstream.chat.android.client.ChatClient

class AppSettings(val context: Context) {

    /**
     * See: [Client Location Settings](https://getstream.io/chat/docs/android/multi_region/?language=kotlin#client-location-settings)
     */
    fun clientLocationSettings() {
        // Set the base URL
        val client = ChatClient.Builder("{{ api_key }}", context)
            .baseUrl("https://chat.stream-io-api.com/")
            .build()
    }
}
